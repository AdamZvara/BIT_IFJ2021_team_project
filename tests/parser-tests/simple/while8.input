require "ifj21"

function foo(i : integer)
	local f : string = "ahoj"..("kek".."kok\n")
	if i > #f then
		write("BIG\n")
	else
		write("SMALL\n")
	end
	local a : integer = 5
	while i > 0 do
		if (i > a) then
			write("i is bigger\n")
			a = i -a
		else
			write("i is smaller\n")
			a = a - i
		end
	i = i - 1
	end
end


foo(5)
