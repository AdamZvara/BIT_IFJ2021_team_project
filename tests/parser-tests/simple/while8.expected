SMALL
i is smaller
i is bigger
i is smaller
i is bigger
i is smaller
